packager/packager.js

```diff
@@ -160,7 +160,13 @@ var server = runServer(options, function() {
   console.log('\nReact packager ready.\n');
 });
 
-webSocketProxy.attachToServer(server, '/debugger-proxy');
+var proxy = webSocketProxy.attachToServer(server, '/debugger-proxy');
+
+function isDebuggerConnected() {
+  // Debugger is connected if the app and at least one browser are connected
+  // to the websocket proxy.
+  return proxy.clientsCount() >= 2;
```

Comment:
If both devices have debug mode enabled, then no, the debugger won't be connected even though two devices are connected to the WS proxy.

The case I can think of where we might have a problem:
1. Developer opens iOS app in debug mode
2. Chrome automatically opens debugger page
3. Developer opens Android app in debug mode
4. Chrome debugger automatically reloads with Android code
5. Developer closes Chrome debugger
6. Developer reloads iOS app
7. Developer expects Chrome to automatically open debugger, but nothing happens

I'm not sure how common a use case this is, but I think to be safe we may be able to grab the client's user agent when they connect and save it in the `clients` list. Will investigate some more tonight.